applications/rag/workloads.tfvars

```diff
@@ -31,70 +30,42 @@ create_gcs_bucket    = true
 # To verify, check that `gcloud storage buckets describe gs://<bucketname>` returns a 404.
 gcs_bucket = "rag-data-<username>"
 
-cloudsql_instance        = "pgvector-instance"
-cloudsql_instance_region = "us-central1" # defaults to cluster_location, if not specified
+# Ensure the instance name is unique to your project.
+cloudsql_instance = "pgvector-instance"
 
 ## Service accounts
 
 # Creates a google service account & k8s service account & configures workload identity with appropriate permissions.
-# Set to false & update the variable `ray_service_account` to use an existing IAM service account.
-create_ray_service_account      = true
 ray_service_account             = "ray-sa"
 enable_grafana_on_ray_dashboard = false
 
 # Creates a google service account & k8s service account & configures workload identity with appropriate permissions.
-# Set to false & update the variable `rag_service_account` to use an existing IAM service account.
-create_rag_service_account = true
-rag_service_account        = "rag-sa"
+rag_service_account = "rag-sa"
 
 # Creates a google service account & k8s service account & configures workload identity with appropriate permissions.
-# Set to false & update the variable `jupyter_service_account` to use an existing IAM service account.
 jupyter_service_account = "jupyter-sa"
 
 ## Embeddings table name - change this to the TABLE_NAME used in the notebook.
-dataset_embeddings_table_name = "googlemaps_reviews_db"
+dataset_embeddings_table_name = "netflix_reviews_db"
 
-## IAP config - if you choose to disable IAP authenticated access for your endpoints, ignore everthing below this line.
-create_brand  = false
-support_email = "<email>" ## specify if create_brand=true
+## IAP Config
+
+# If you disable authenticated access for your endpoints, ignore everthing below this line.
```

Comment:
Easy to miss. Consider a box comment:

```
##########
# comment  #
##########
```

Reply:
Also, this a chicken-and-egg, because enabling IAM happens below. Maybe just remove the comment? Is it not clear that the parameters are only needed if `*_add_auth=true` for each component below?

Reply:
tried to make the wording a bit clearer, but it wasn't clear to me